www/caddy/src/opnsense/mvc/app/models/OPNsense/Caddy/Caddy.xml

```diff
@@ -10,6 +10,7 @@
             </enabled>
             <TlsEmail type="EmailField">
                 <ValidationMessage>Please enter a valid email address.</ValidationMessage>
+                <Required>Y</Required>
```

Comment:
I think this will break the basic validation for empty setups, which is probably not desirable. For static options a require without a provided default is useless. And if there is no good default (like in this case) there is no point to require it out of the box (it cannot be guessed).

Reply:
I understand. It's really an annoying edge case. One can only hope then, that the user will fill it out.

Reply:
Well, you could add a constraint that if enabled it requires an email. If upstream caddy fails if email is not set consider bootstrapping a value from the template. Some people will not like to add their mail just because and input garbage anyway.

Reply:
Caddy wont refuse to start. It will just not issue ZeroSSL certificates anymore and always use Let's Encrypt and log errors. It's all transient though. So updating the help text should be enough.

Reply:
Sounds like a plan. Laying this out in more detail on the docs would also be beneficial.. some people don't "see" help texts.